velero-plugin-for-aws/object_store.go

```diff
@@ -35,6 +35,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/aws/aws-sdk-go/service/s3/s3manager"
 	"github.com/pkg/errors"
+	"github.com/prometheus/common/log"
```

Comment:
I think a new logger (prometheus common log) is being introduced here. Velero appears to use logrus. Intentional?

Reply:
not intentional, IDE screwed me :D I will post an update to this.

Reply:
updated, PTAL